src/rtkpos.c

```diff
@@ -2474,12 +2551,18 @@ extern void rtkinit(rtk_t *rtk, const prcopt_t *opt)
 extern void rtkfree(rtk_t *rtk)
 {
     trace(3,"rtkfree :\n");
-    
+
     rtk->nx=rtk->na=0;
     free(rtk->x ); rtk->x =NULL;
     free(rtk->P ); rtk->P =NULL;
     free(rtk->xa); rtk->xa=NULL;
     free(rtk->Pa); rtk->Pa=NULL;
+
+    if ( rtk->glo_IFB != NULL ) {
+
+        glo_IFB_free((glo_IFB_t *) rtk->glo_IFB);
```

Comment:
You should get rid of the cast here by adding a type declaration to the rtk_t->glo_IFB.